.github/workflows/bump-packages.yaml

```diff
@@ -10,25 +10,39 @@ jobs:
     name: Bump packages
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/create-github-app-token@v1
```

Comment:
Here we can't use the [setup-bot-token](https://github.com/mongodb-js/devtools-shared/blob/main/actions/setup-bot-token/action.yml) action because it's in the same repo and we must checkout it first to be able to use it.